src/zerocoin_v3.cpp

```diff
@@ -51,6 +51,17 @@ static bool CheckZerocoinSpendSerialV3(
 	return true;
 }
 
+bool IsSigmaAllowed()
+{
+	LOCK(cs_main);
```

Comment:
After we merge this one, this lock is not necessary any more right? https://github.com/zcoinofficial/zcoin/pull/373/files

Reply:
If we continue with the way in that PR, yes.